### PR TITLE
spg Chapter 8

### PR DESCRIPTION
**Pull request by SimonGross**
Chapter 8 - now with a slight Christmasy theme

### CODE REVIEW

File: src/com/datacash/training/answers.txt [hunk 1]
```diff
@@ -0,0 +1,38 @@
+1) See SantasLists.java etc.
+
+2) The namespace is a declarative region. By partitioning the namespace you can have multiple classes with different names so long as they are in different parts of the namespace.
```
Comment (codavex) [inline src/com/datacash/training/answers.txt:3]: ...multliple classes with the _same_ name...

File: src/com/datacash/training/qpack/Child.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+// A child in Santa's list. Have they been naughty or nice this year?
+public class Child {
+
+	private ChildStatus status;
+	private int id;
+
+	// Assign a status based on child's naughtiness level
+	public Child( int id, double naughtiness ) {
+		if ( naughtiness > 0.5 ) {
```
Comment (codavex) [inline src/com/datacash/training/qpack/Child.java:12]: Sorry, surely this check needs carried out twice, it says so in the song

File: src/com/datacash/training/answers.txt (hunk 1 above)
Comment (SimonGross) [inline src/com/datacash/training/answers.txt:3]: Oops. That's what I wanted to write, but I was pretty tired.

File: src/com/datacash/training/answers.txt [hunk 3]
```diff
@@ -0,0 +1,38 @@
+1) See SantasLists.java etc.
```
Comment (codavex) [inline src/com/datacash/training/answers.txt:1]: Bonus points for the Santa theme. Very Christmas-y. :+1: :christmas_tree: :santa: :snowman: :gift:

**Pull request closed without merging**